pkg/gateway/api/api_grpc.go

```diff
@@ -1218,7 +1218,9 @@ func getEnvironmentAPIKeyFromCache(
 }
 
 func checkEnvironmentAPIKey(environmentAPIKey *accountproto.EnvironmentAPIKey, role accountproto.APIKey_Role) error {
-	if environmentAPIKey.ApiKey.Role != role {
+	// TODO: Fix the condition after migration
+	// The role must be UNKNOWN or SDK_CLIENT until the migration is done
+	if environmentAPIKey.ApiKey.Role == accountproto.APIKey_SDK_SERVER {
```

Comment:
As described in the comment, once the migration is done, I will revert the condition.